### ISSUE TITLE
Audio continues even without user input

### ISSUE THREAD

**Issue by BestPresident** [ruffle-rs/ruffle#1413]
I've been testing quite a few SWF files from my site. This is a bug I've noticed that seems present in many animations.
In these SWFs the audio should start/continue only after user input (clicking a button on the screen). 

Here are 4 games that have this problem:
[alfy.zip](https://github.com/ruffle-rs/ruffle/files/5450408/alfy.zip)

**Comment by Herschel** [comment]
Thanks for the report! Are you testing on web or desktop? I'm not sure I'm able to repro the issue, assuming it should occur if I don't play the play button on the first screen. It's possible this was fixed by #1395 or #1405. Could you try your SWF on the demo page at https://ruffle.rs/demo/ ?

**Comment by BestPresident** [comment]
Thanks for the reply! I'm testing on my site but maybe the core isn't the latest version. I tried them on the demo site, and they work better but the first sequence is still buggy (audio starts without input). If you try alfycamping.swf you'd see what I mean.

**Comment by BestPresident** [comment]
I've updated the core and tried them a few times and they seem to be working pretty well. My bad!

**Comment by Toad06** [comment]
The issue remains however, if you switch between tabs while the voice is speaking. Probably related to #347 though.

**Comment by Herschel** [comment]
Yes, in that case it's related to #347 (the player isn't updating while offscreen, so it has no way to tell the sound to stop). Closing in favor of #347.
